vm/src/stdlib/os.rs

```diff
@@ -1191,6 +1191,27 @@ fn os_chmod(
     Ok(())
 }
 
+#[cfg(unix)]
+fn os_execv(
+    path: PyStringRef,
+    argv_list: PyIterable<PyStringRef>,
+    vm: &VirtualMachine,
+) -> PyResult<()> {
+    let path = ffi::CString::new(path.as_str()).unwrap();
```

Comment:
how CPython work when path includes null character? Maybe there is some error then we'd better to raise here too.

Reply:
CPython raises ValueError for null-embeded path, and same exception is also raised for empty tuple/lists. These two exceptions should be taken care of.